source/includes/_run.md

```diff
@@ -1,30 +1,64 @@
-# Run irohad locally
+# Run the daemon (irohad) 
 
-## Generate genesis block
+## Create genesis block
 
-1. Create peers.list file containing the peers' addresses. For the local node suffice to add only single peer:
-```bash
-$ echo 'localhost:10001' > peers.list
-```
+A genesis block is the first block in the chain. <br>
+<b>Important:</b> each peer in the initialized network should have exactly the same genesis block, so create it once and distribute across peers. This block may have <b>any data</b>, as no-one validates it in the peer network.
+
+### Using iroha-cli
+
+#### Create list of peers
+Create peers.list file containing network addresses of peers' internal ports in the network (a single peer address goes on new line. If you want to run Iroha on single peer, it is sufficient to mention only localhost: <br> `$ echo 'localhost:10001' > peers.list`
+
+As an example, for 4 peers the file will look like the following:
+<ul>
+  <li>10.128.13.1:10001</li>
+  <li>10.128.13.2:10001</li>
+  <li>10.128.13.3:10001</li>
+  <li>10.128.13.4:10001</li>
+</ul>
+
+<aside class="notice">
+Internal port is used for communication of internal components. Each peer potentially may have different, adjustable internal port numbers.
+</aside>
+
+#### Use iroha-cli
+Generate genesis block using iroha-cli (see shell command on the right side).
+
+Iroha-cli tool is used to generate keypairs for every peer, mentioned in the peers list. This is done only for convenience — in production environment please consider using manual or more trusted approach.
+
+> Generating genesis block
 
-2. Generate genesis block using iroha-cli:
 ```bash
-$ iroha-cli --genesis_block --peers_address peers.list
+$ iroha-cli --genesis_block \ 
+--peers_address peers.list
 ```
+Executing this command will result in generation of following files:
+
+* `admin@test.priv` and `admin@test.pub` — keypair for user with admin and asset creator rights
+* `test@test.priv` and `tes@test.pub` — keypair for user with user rights
```

Comment:
Missed 't' at first part of second file name.

Reply:
ok